.devcontainer/devcontainer.json

```diff
@@ -54,7 +54,7 @@
 		}
 	},
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "pip install -r requirements-dev.txt && pip install -e src",
+	"postCreateCommand": "pip install -r requirements-dev.txt && pip install -e src/backend && pip install -e src",
```

Comment:
It feels weird that we have to install both. What happens if we only install src/backend?

Reply:
nothing, guess I never tried it lol. it works 🤣